client/app/scripts/superdesk/directives/sdTypeahead.js

```diff
@@ -147,9 +147,15 @@ define(['angular'], function(angular) {
                         if (visible || scope.alwaysVisible) {
                             $list.show();
                             scope.hide = false;
+                            $timeout(function() {
+                                $input.focus();
+                            });
```

Comment:
would be good to add `0, false` to `$timeout`, there is no need to trigger digest cycle for this